README.md

```diff
@@ -2,4 +2,12 @@
 
 #### 1/15/2021
 
-##### diegodiaz/calculator_ui
\ No newline at end of file
+##### diegodiaz/calculator_ui
```

Comment:
please add a brief explanation about what happened on this commit

Reply:
Updated!